djangobaseview/static/js/guestbook/store/GreetingStore.js

```diff
@@ -0,0 +1,37 @@
+define([
+	"dojo/request",
```

Comment:
- hãy sắp xếp lại import theo thứ tự
  - http://192.168.1.248/redmine/projects/rakumo/wiki/Rakumo_source_code_review_checklist
  - dojo / _base / *
  - dojo / *
  - dijit / *
  - dojox / *
  - fasti / common ...
  - fasti / board ...
  - (Modules that are not required as an argument, but need for data-dojo-type in template)